pkgs/top-level/all-packages.nix

```diff
@@ -8377,6 +8377,8 @@ with pkgs;
 
   freetube = callPackage ../applications/video/freetube { };
 
+  invidtui = callPackage ../applications/video/invidtui { };
+
```

Comment:
```suggestion
```
New top-level packages should go to `pkgs/by-name` https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md